src/modules/pos/validator/components/ChangeCommission/Form/ChangeCommissionForm.test.js

```diff
@@ -158,22 +158,33 @@ describe('ChangeCommissionForm', () => {
       fireEvent.change(input, { target: { value } });
       fireEvent.click(button);
 
+      const feeToken = { availableBalance: 40000000, chainID: '04000000', symbol: 'LSK' };
+
```

Comment:
NRTTL:
When using custom fee it can be displayed different in first step from the second step, feels like the user could feel tricked by this.  

https://github.com/LiskHQ/lisk-desktop/assets/8784876/dafc4395-eb70-47b2-8b36-3c185202773d